guides/source/active_record_migrations.md

```diff
@@ -436,8 +436,17 @@ change_column_default :products, :approved, false
 This sets `:name` field on products to a `NOT NULL` column and the default
 value of the `:approved` field to false.
 
-TIP: Unlike `change_column` (and `change_column_default`), `change_column_null`
-is reversible.
+Additionaly, for `change_column_default`, you can pass a Hash which defines the
+changes. This will make the command reversible. The command above can be
+rewritten as:
+
+```ruby
+change_column_default :products, :approved, from: true, to: false
+```
+
+TIP: `change_column` command is always irreversible, `change_column_null` is
+always reversible, and `change_column_default` is only reversible if using
```

Comment:
Not in scope of this PR, but would be nice if we could extract keys from `change_column` command, and normalize them to reversible migrations as long as all keys are reversible. E.g. combining `null` and `default` often goes together in a migration that sets null to false:

```ruby
change_column 'foo', null: false, default: { from: nil, to: 'bar' }
```

Would be interpreted as:

```ruby
change_column_default 'foo', from: nil, to: 'bar'
change_column_null 'foo', false
```

Above order is important, if null is disallowed the default must be changed first, if null is allowed the default must change last.